info/v1/container.go

```diff
@@ -415,6 +415,9 @@ type FsStats struct {
 	// Number of bytes available for non-root user.
 	Available uint64 `json:"available"`
 
+	// Number of Inodes
+	Inodes uint64 `json:"inodes"`
```

Comment:
we need to potentially treat this and inodes_free to be optional fields so we can distinguish the field as being set versus unset in cases where its not knowable.